qiita_db/support_files/patches/python_patches/66.py

```diff
@@ -1,15 +1,30 @@
-# August 6, 2018
-# Create parameters for the ssh/scp remote file upload commands
-
-
 from json import loads, dumps
-
 from qiita_db.sql_connection import TRN
 from qiita_db.software import Software, Command
 from qiita_db.exceptions import (QiitaDBError, QiitaDBDuplicateError)
 from qiita_db.util import convert_to_id
+from qiita_db.study import Study
+from re import sub
 
+# August 31, 2018
```

Comment:
This patch needs to be moved to its own file.

Reply:
I can do this in my PR as I'm fixing conflicts; however, in the past we have decided to have only one patch per release cycle ... thus; the idea of having in the same patch